libs/vm_monitor/src/lib.rs

```diff
@@ -19,11 +24,14 @@ pub mod cgroup;
 pub mod filecache;
 pub mod runner;
 
-/// Arguments (previously command-line) to configure the monitor.
-#[derive(Debug)]
+/// Arguments to configure the monitor.
+#[derive(Debug, Parser)]
 pub struct Args {
+    #[arg(short, long)]
     pub cgroup: Option<String>,
+    #[arg(short, long)]
     pub pgconnstr: Option<String>,
+    #[arg(short, long)]
     pub addr: String,
```

Comment:
Should have some brief comments here about what the arguments are for. (and then, they'll get included in the `-h`/`--help` output!)